KS-3-5 задача 5

VVShchegolev opened the pull request:
Распарсил строку – разбил ее на 3 составляющих

Commit:
KS-3-5 задача 5

ievetrovdev commented on src/main/kotlin/Lesson_3_Ex_5/Lesson_3_Ex_5.kt:
```diff
@@ -0,0 +1,21 @@
+    val moveString = "D2-D4;0"
+
+//    Разбираем строку на части по ";"
+    val parts = moveString.split(";")
```
Верное направление! Однако, сплит умеет принимать несколько параметров. Можно существенно сократить код

VVShchegolev committed:
Update KS-3-5 Задание 5

Что было исправлено:
Добавил в split нужные значения, и, этим самым сократил код)
